index.html

```diff
@@ -1,53 +1,124 @@
+</td>
+<td style="width:45%">
+<p>
+<b>Dafny</b> is a verification-aware programming language that has native support for recording specifications and is equipped with a static program verifier. By blending sophisticated automated reasoning with familiar programming idioms and tools, Dafny empowers developers to write provably correct code (w.r.t. specifications),
+while also compiling Dafny code to familiar development environments such as C#, Java, JavaScript and Go (with more in progress, such as Python) so Dafny can integrate with your existing workflow.
+Dafny makes rigorous verification an integral part of development thus reducing costly late-stage bugs that may be missed by testing.
+</p>
+</td>
+<td style="width:40%">
+<b>Quick Links:</b>
+<ul>
+<li> <a href="https://github.com/dafny-lang/dafny/wiki/INSTALL"><b>Installation</b></a> 
+     (or a <a href="https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode">VSCode plugin for Dafny</a>)
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef">Dafny Reference Manual and User Guide</a>
+<li> <a href="https://dafny.org/dafny/toc">Dafny Resources for Users</a>
+<li> <a href="https://github.com/dafny-lang/dafny">Dafny GitHub project (for developers of the Dafny tools themselves)</a>
+</ul>
+</td>
+
+
+</tr>
+</table>
+
+<table style="width:100%">
+<td style="width:65%" valign=top>
+<p>
+In addition to a verification engine to check implementation against specifications, the Dafny ecosystem includes several compilers, plugins for common software development IDEs, a LSP-based Language Server, a code formatter, a reference manual, tutorials, power user tips, books, the experiences of professors teaching Dafny, and the accumulating expertise of industrial projects using Dafny.
+Example Dafny code in a Visual Studio IDE is shown to the right.
+
+</p>
+
+<p>Dafny has support for common programming concepts such as </p>
+<ul>
+<li> mathematical and bounded integers and reals, bit-vectors, classes, iterators, arrays, tuples, generic types, refinement and inheritance
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-inductive-datatypes">inductive datatypes</a> that can have methods and are suitable for pattern matching,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-co-inductive-datatypes">lazily unbounded datatypes</a>,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-subset-types">subset types</a>, such as for bounded integers,
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-lambda-expressions">lambda expressions</a> and functional programming idioms,
+<li> and <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-collection-types">immutable and mutable data structures</a>.
+</ul>
+
+<p>Dafny also offers an extensive toolbox for mathematical proofs about software, including</p>
+<ul>
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-forall-statement">bounded and unbounded quantifiers</a>,
```

Comment:
Perhaps https://dafny.org/dafny/DafnyRef/DafnyRef#sec-quantifier-domains would be a better generic starting point for quantification in general, since `forall` statements aren't used nearly as much as `forall/exists` expressions in my experience?

Although now that I look at it I wish that section ended with links to specific features that use quantification, feel free to fix that too.